packages/shopify-codemod/transforms/iife-to-ternary-expression.js

```diff
@@ -0,0 +1,48 @@
+export default function iifeToTernaryExpression({source}, {jscodeshift: j}, {printOptions = {}}) {
+  function isImmediatelyInvokedFunctionExpression(expressionPath) {
+    let expression = expressionPath.node.expression
```

Comment:
You can use `const` here.  This is probably causing a CI failure.

I recommend adding adding a Git pre-push hook to prevent minor stuff reaching review.  Steps for that: 

* Go to your `shopify/javascript` dir
* `echo 'cd packages/shopify-codemod && npm test' > .git/hooks/pre-push`

This'll run the codemod tests, every time you try to git push, and back out if the tests/linting fail.